packages/kcmsf/src/pages/result.tsx

```diff
@@ -180,32 +180,26 @@ const PreResultColum = (props: { match: PreMatch }) => {
       ),
     [props.match]
   );
-  const leftGoalTimeSeconds = useMemo(() => {
-    if (!leftResult) {
-      return "";
-    } else {
-      if (leftResult.goalTimeSeconds)
-        return parseSeconds(leftResult.goalTimeSeconds);
-      return "フィニッシュ";
-    }
-  }, [leftResult]);
-  const rightGoalTimeSeconds = useMemo(() => {
-    if (!rightResult) {
-      return "";
-    } else {
-      if (rightResult.goalTimeSeconds)
-        return parseSeconds(rightResult.goalTimeSeconds);
-      return "フィニッシュ";
-    }
-  }, [leftResult]);
   return (
     <>
       <Table.Td className="td">{props.match.leftTeam?.teamName}</Table.Td>
       <Table.Td className="td">{leftResult?.points}</Table.Td>
-      <Table.Td className="td">{leftGoalTimeSeconds}</Table.Td>
+      <Table.Td className="td">
+        {leftResult
+          ? leftResult.goalTimeSeconds === null
```

Comment:
```suggestion
          ? leftResult.goalTimeSeconds == null
```
`goalTimeSeconds`は`number | undefined`なので、`undefined`の時に`null`と厳密比較すると`false`になってしまいます。
![image](https://github.com/user-attachments/assets/e3e2fa34-1d48-42a8-9583-e5e9c95efd36)

Reply:
そうするとこれってエディタ側のミスの可能性ある?
![image](https://github.com/user-attachments/assets/75e50bb7-db7d-4ddc-b564-3d343450ddab)

Reply:
ありゃ

Reply:
レスポンスデータをそのまま引き回してるから`number | null`であってるのか

Reply:
じゃあなんで`NaN:NaN`になるんですか？？？

Reply:
mockのミスで`null`になるべきが`undefined`(=フィールドがない)になっていたようです
つまり以前のものが正しいです gomenne

Reply:
OK